salt/modules/git.py

```diff
@@ -1852,7 +1852,7 @@ def diff(
     output_encoding=None,
 ):
     """
-    .. versionadded:: 2015.8.12,2016.3.3,2016.11.0
+    .. versionadded:: 2015.8.12, 2016.3.3, 2016.11.0
```

Comment:
```suggestion
    .. versionadded:: 2015.8.12/2016.3.3/2016.11.0
```

When introducing spaces between versions, this looks like the `.. versionadded::` directive ends up breaking the formatting. A space looks like the expected end of the version(s), so this seems to be why commas-without-spaces were used. Though, it makes the readability a little off. I'm wondering if comma-separated versions would be more presentable with slashes?

Otherwise we could revert to the commas without spaces:

```rst
.. versionadded:: 2015.8.12,2016.3.3,2016.11.0
```

> There are more instances of this throughout this PR, but I am only flagging one since there are two approaches to resolving: going with the current standard of commas without spaces, or with slashes for better readability.

Reply:
We voted to remove the space, and revert to comma separated values as a follow-up application from the docstrings tool.